client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioningInfo/ContactDetails.tsx

```diff
@@ -28,6 +28,19 @@ const ContactDetails = (props: Props) => {
 
     const { isGroupedContact } = GetGroupedInvestigationsIds();
 
+    const highestContactType = interactedContact.contactEvents.reduce((prev, current) => {
+        if(current.contactType === 1)  {
```

Comment:
you should save the contact type in const with meaningful name

Reply:
fixed [here](https://github.com/CoronaTeamMOD/CoronaInvestigation/pull/1384/commits/eab3efc89f4a2466c96ab9b7ad6b816da877fe97)